tests/model_test.cpp

```
@@ -0,0 +1,211 @@
+TEST_F(OpenVinoNetTest, AsyncInferFailsWithEmptyImage) {
+    bool invoked = false;
+    net_.async_infer(Image{}, [&](auto result) {
+        invoked = true;
+        ASSERT_FALSE(result.has_value());
+        EXPECT_EQ(result.error(), "Empty image mat");
+    });
+    EXPECT_TRUE(invoked);
+}
+
+TEST_F(OpenVinoNetTest, ConfigureSuccessWithValidModelPath) {
+    if (!HasValidModel())
```

Clang (14.0.6)

[error] statement should be inside braces

(google-readability-braces-around-statements,readability-braces-around-statements,-warnings-as-errors)